msal/src/test/res/raw/native_auth_native_only_test_config.json

```diff
@@ -0,0 +1,10 @@
+{
+  "client_id" : "0e565df2-c350-47f2-9cd4-c6c7a464b8bf",
+  "authorities" : [
+    {
+      "type": "CIAM",
```

Comment:
Do you want AAD_NA instead?

Reply:
Is this addressed?

Reply:
This must be CIAM. I checked this in the documentation for private preview.